src/index.js

```diff
@@ -1,4 +1,5 @@
 /* @flow */
+import "babel-polyfill";
```

Comment:
I'm not an expert, but it seems like the kinto-http.js artifacts did not include async/await code, but were already transformed into something that relied on a `regeneratorRuntime`. This `regeneratorRuntime` was provided by `babel-polyfill` or the "regenerator runtime", whatever that was.